server/config.py

```diff
@@ -56,6 +56,7 @@ class EnvironmentConfig:
 
     # Image sources
     MAPILLARY_API = {'base': 'https://a.mapillary.com/v3/', 'clientId': os.getenv('MAPILLARY_API_KEY', None)}
+    MAPILLARY_TIME_CORRELATION = os.getenv('MAPILLARY_TIME_CORRELATION', 60)
```

Comment:
Just a note, you want the default to be 60, while we can set it to be 600 in the server config.

Right?